torch/distributed/fsdp/_fully_shard/_fsdp_collectives.py

```diff
@@ -203,7 +203,8 @@ def all_gather_copy_in_cuda(
 def split_with_sizes_copy(
```

Comment:
If all you are doing is forwarding args wouldn't
`split_with_sizes_copy = torch.split_with_sizes_copy`
be sufficient? Do you explicitly want to ignore the return type here?